src/status_im/ui/screens/wallet/settings/views.cljs

```diff
@@ -26,8 +26,10 @@
             visible-tokens [:wallet/visible-tokens-symbols]]
     [react/view (merge components.styles/flex {:background-color :white})
      [status-bar/status-bar {:type :modal-wallet}]
-     [toolbar/toolbar #_{} {:style wallet.styles/toolbar}
-      [toolbar/nav-text {:style               {:color :white}
+     [toolbar/toolbar {:style wallet.styles/toolbar}
+      [toolbar/nav-text {:handler             #(do (re-frame/dispatch [:update-wallet])
+                                                   (re-frame/dispatch [:navigate-back]))
```

Comment:
Why `navigate-back` ?

Reply:
what do you mean? it was navigate back by default, i added update-wallet

Reply:
Ah of course!